### PR TITLE
people component

### LINKED ISSUE #2: Integrate People Backend

### LINKED ISSUE #6: Add People Config Node in UI

### PR DESCRIPTION
**Pull request by michal367**
closes #2 
closes #6

### CODE REVIEW

File: people_server/src/peopleServer.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        return -1;
+    }
+
+    const std::string address = "localhost";
+    const int port = 5672;
+    const std::string user = "myuser";
+    const std::string password = "mypassword";
+    const std::string exchange = "ImageFinder";
+    const std::string queue_name = "image_finder.people";
+    const std::string results_queue_name = "image_finder.results";
```
Comment (Derstilon) [inline people_server/src/peopleServer.cpp:29]: This should be loaded from the ini file
Most recommended way I found is [simpleini](https://github.com/brofield/simpleini)

File: people_server/src/SimplePocoHandler.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#include <memory>
+#include <amqpcpp.h>
+
+
+class SimplePocoHandlerImpl;
+class SimplePocoHandler: public AMQP::ConnectionHandler
+{
+public:
+    static constexpr size_t BUFFER_SIZE = 8 * 1024 * 1024; //8Mb
+    static constexpr size_t TEMP_BUFFER_SIZE = 1024 * 1024; //1Mb
```
Comment (Derstilon) [inline people_server/src/SimplePocoHandler.h:14]: Did you handle error for situation when buffer size would be too low?

File: Client/components.json [hunk 3]
```diff
@@ -67,5 +67,10 @@
         "no smiles": ["float", "smiles"],
         "comparator": ["choice_propagate", ">", ">=", "<", "<=", "=="],
         "threshold": ["float", "equal"]
+    },
+    "People": {
+        "hasPeople": ["choice_propagate", "true", "false"],
+        "minPeople": ["int", "true"],
+        "maxPeople": ["int", "true"]
```
Comment (Derstilon) [inline Client/components.json:74]: You need to either change the input's label of "int" or rewrite if handler to accept both your type with parameter "true" and another type with min-max range values

**Review by Derstilon — CHANGES_REQUESTED** [review]
Your changes work but your changes break the weather component because you changed how int input works
image

File: people_server/src/SimplePocoHandler.h (hunk 2 above)
Comment (michal367) [reply people_server/src/SimplePocoHandler.h:14]: The code is from here:
https://github.com/RPG-18/rabbitmq-cpp-tutorials/blob/master/src/SimplePocoHandler.h
https://github.com/RPG-18/rabbitmq-cpp-tutorials/blob/master/src/SimplePocoHandler.cpp

Buffer is limited by 8Mb, but I think that it is very big number. I think it is out of scope of our app anyway.

**Review by Derstilon — APPROVED** [review]

**Pull request merged**